py/client-ticking/demo_simple.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+import pydeephaven as pyd
+import pyarrow as pa
+import time
+from pydeephaven_ticking.table_listener import TableListener, TableUpdate, TableListenerHandle, listen
```

Comment:
This works, but I think it adds a bit burden to the users. Suggestions:
In the `py/client/__init__.py` add the following
```
try:
    import pydeephaven_ticking as table_listener
except ImportError:
    pass
```
then this line can be changed to
`from pydeephaven.table_listener import TableListener, ...`

Reply:
ok I have done the import thing for __init__.py

Reply:
> Are all of these datastructures now 100% consistent with the server
They are "qualitatively" consistent but they are not 100% consistent. The differences are:

1. The client routines return pyarrow arrays; the server routines return numpy arrays.
2. The client cannot add a second listener to the same subscription
3. There's no `replay` flag. The first message of a subscription contains adds for all the existing rows in the tables. This is tantamount to forcing `replay` to `true`.